### ISSUE TITLE
Future Visual Prediction Loss

### ISSUE THREAD

**Issue by M-o-magic** [alibaba-damo-academy/RynnVLA-001#15]
关于论文中提到的Future Visual Prediction Loss，代码中好像并不存在这部分代码？

**Comment by M-o-magic** [comment]
我找到对应的Future Visual Prediction Loss代码了，但是我有一个新问题，shift_logits中66540哪些是images相关的token？我用logits_max = torch.argmax(shift_logits, dim=-1) ，再用vqgan解码logits_max ，可视化效果非常糟糕。。

(Pdb) shift_logits.shape
torch.Size([11231, 65540])
(Pdb) shift_labels.shape
torch.Size([11231])
